src/views/Admin/Modules/Records/List.vue

```diff
@@ -456,297 +67,19 @@ export default {
       }
     },
 
-    fields () {
-      let fields = []
-      if (this.module) {
-        const pre = []
-        const post = []
-
-        // Use first five fields from the module.
-        fields = this.module.fields.slice(0, 5)
-
-        pre.push({ key: 'selectable', label: '', tdClass: 'selector', thClass: 'selector' })
-
-        const configured = fields.map(mf => ({
-          key: mf.name,
-          label: mf.label || mf.name,
-          moduleField: mf,
-          sortable: !mf.isMulti && mf.isSortable,
-          tdClass: 'record-value',
-        }))
-
-        post.push({ key: 'actions', label: '', tdClass: 'actions' })
-
-        return [
-          ...pre,
-          ...configured,
-          ...post,
-        ]
-      }
-      return fields
-    },
-
-    getFilterButtonInfo () {
-      let label = this.$t('block.recordList.filter.label')
-      let variant = 'light'
-
-      if (this.filterFields.active) {
-        label = `${label} ${this.$t('general.label.active')}`
-        variant = 'primary'
-      }
-
-      return { label, variant }
-    },
-
-    hasPrevPage () {
-      return this.filter.prevPage
-    },
-
-    hasNextPage () {
-      return this.filter.nextPage
-    },
-
-    getPagination () {
-      const { count = 0, page = 1 } = this.pagination
-      const { limit = 20 } = this.filter
-
-      return {
-        from: ((page - 1) * limit) + 1,
-        to: Math.min((page * limit), count),
-        page,
-        perPage: limit,
-        count,
-      }
-    },
-  },
-
-  watch: {
-    query: throttle(function (e) {
-      this.filter.query = queryToFilter(this.query, '', this.module.filterFields(this.module.fields.slice(0, 5)))
-      this.pullRecords(true)
-    }, 500),
   },
 
   created () {
-    this.pullRecords(true)
+    const options = {
+      moduleID: this.$attrs.moduleID,
+      fields: this.allModuleFields,
+      customFields: true,
+      rowViewUrl: 'admin.modules.record.view',
+      rowEditUrl: 'admin.modules.record.edit',
+      rowCreateUrl: 'admin.modules.record.create',
+    }
```

Comment:
Also add options that disable view icon(eye), reminders(bell) and cloning(copy icon). Also enable exporting